doc: update readme for CPC (#182)
openjs-foundation/cross-project-council: README.md

# The OpenJS Foundation Cross Project Council

The OpenJS Foundation Cross Project Council (CPC) is the technical governing body of the OpenJS Foundation. It is described in the [CPC Charter](./CPC-CHARTER.md)

## CPC Scope

**The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**

The CPC exercises autonomy in managing its responsibilities and seeks agreement from the OpenJS Foundation Board on any change to the scope of those responsibilities.

### OpenJS Fondation Projects

* To be filled in

> ## OpenJS Foundation Members

Reviewer: ```suggestion
## CPC Members
```

> ### Voting Memberss

Reviewer: ```suggestion
### Voting Members
```

> 

Reviewer: * @mcollina - Matteo Collina <matteo.collina@gmail.com> (he/him)

Reviewer 2: add
```suggestion
- **WebdriverIO**: Christian Bromann (@bromann)
```

Reviewer 3: @MylesBorins could you clarify the intent/reason of including this list? Given the context of it as you've written compared to the comments on it, I'm not sure if I'm missing something.

# The OpenJS Foundation Cross Project Council

The OpenJS Foundation Cross Project Council (CPC) is the technical governing body of the OpenJS Foundation. It is described in the [CPC Charter](./CPC-CHARTER.md)

## CPC Scope

> **The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**

Reviewer 4: Oxford comma?

Reviewer 4: s/ensuring/insuring/?

> **The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**

Reviewer 4: I'll try to come up with something for this sentence, but if not, two things:

* Please remove the bold text? Pretty please?
* This is a run-on sentence and difficult to understand. It should be re-worded and/or split into more than one sentence.

Reviewer 4: (Deleted my previous comments because I think they're wrong because I was misreading the sentence because yikes!)

Reviewer 4: I think this sentence should be removed or else completely rewritten with a careful eye towards being obviously meaningful. As it stands, it is difficult to see anything meaningful in it. Let's break this sentence down:

> The CPC ensures collaboration is the driving principle within a Project.

*The* driving principle? I'm not even sure it's possible for the CPC to do that. What do they do? Check on the project? Or actively manage the project if "collaboration" is not "the driving principle"? How does one evaluate whether collaboration is "the driving principle" of anything?

> The CPC ensures collaboration is the driving principle between OpenJS Foundation Projects.

What does that even mean? What is "the driving principle between OpenJS Foundation Projects"? There aren't driving principles between entities. That's not a thing.

> The CPC ensures collaboration is the driving principle between OpenJS Foundation Projects and the broader community.

Same as above, but even more so.

Reviewer 4: Maybe something more like this?

```suggestion
The CPC primary role is to foster an environment of collaboration. That environment extends within and between OpenJS Foundation Projects. It also extends between the Projects and the larger community.
```

*How* the CPC will do that is left as an exercise to the CPC itself, I suppose, but I think that sentence at least as meaning. For example, it is much easier for me to understand what a collaborative-environment-between-projects looks like than collaboration-as-driving-principle-between-projects.

> **The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**

Reviewer 5: The second bit is part of the larger phrase "between OpenJS Foundation Projects and the broader community."

So three scopes:
1. within a Project
2. between OpenJS Foundation Projects
3. between OpenJS Projects and the broader community

It ends up sounding a little odd if not read with breaks for each of those scopes, but is fully valid. Make sense?

> **The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**

Author: This section is no longer included anyway 👍🏽

- **Appium**: Dan Graham (@dpgraham) & Jonah Stiennon(@Jonahss)
- **Dojo**: Matt Gadd (@matt-gadd) & Dylan Schiemann (@dylans)
- **jQuery**: Dave Methvin (@dmethvin) & Timmy Willison (@timmywil)
- **Node.js**: Matteo Collina (@mcollina ) & Joe Sepi (@joesepi)
- **webpack**: Sean Larkin (@TheLarkInn) & Even Stensberg (@evenstensberg)

#### Growth & At Large Project Representatives

According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the four individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.

- **Architect**: Kris Borchers (@kborchers)
- **messageformat**: Eemeli Aro (@eemeli)
- **Mocha**: Christopher Hiller (@boneskull)
- **webhint**: Antón Molleda (@molant)

> ### Regular Members

Reviewer 6: To be filled in ?

Reviewer 7: @evenstensberg the plan is to  have people PR themselves in as regular members after this lands.

Reviewer 3: Should be empty for now 👍

Author: I'm going to comment this out until we have members to add

> ### Observers

Reviewer 6: Same here

Reviewer 3: Should also be empty for now 👍

Author: - **webhint**: Antón Molleda (@molant)

# The OpenJS Foundation Cross Project Council

The OpenJS Foundation Cross Project Council (CPC) is the technical governing body of the OpenJS Foundation. It is described in the [CPC Charter](./CPC-CHARTER.md)

## CPC Scope

The CPC's primary role is to foster an environment of collaboration. That environment extends within and between OpenJS Foundation Projects. It also extends between the Projects and the larger community.

The CPC exercises autonomy in managing its responsibilities and seeks agreement from the OpenJS Foundation Board on any change to the scope of those responsibilities.

### OpenJS Fondation Projects

#### Impact Project

* Appium
* Dojo
* jQuery
* Node.js
* webpack

#### Growth Projects

* architect
* Intern
* Mocha
* Node-Red
* WebdriverIO
* webhint

#### At-Large Projects

* ESLint
* Esprima
* Express
* Grunt
* HospitalRun
* Interledger.js
* JerryScript
* Libuv
* Lodash
* Marko

> * Messageformat

Reviewer 8: ```suggestion
* messageformat
```

* Moment
* PEP
* QUnit

#### Emeritus Projects

* Chassis
* jQuery Mobile
* jQuery UI
* RequireJS
* Sizzle

## CPC Members

### Voting Members

#### Impact Project Representatives

Each Impact Project may appoint 2 representatives to the CPC as outlined in the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members)

- **Appium**: Dan Graham (@dpgraham) & Jonah Stiennon(@Jonahss)
- **Dojo**: Matt Gadd (@matt-gadd) & Dylan Schiemann (@dylans)
- **jQuery**: Dave Methvin (@dmethvin) & Timmy Willison (@timmywil)
- **Node.js**: Matteo Collina (@mcollina ) & Joe Sepi (@joesepi)
- **webpack**: Sean Larkin (@TheLarkInn) & Even Stensberg (@evenstensberg)

#### Growth & At Large Project Representatives

> According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the four individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.

Reviewer 8: ```suggestion
According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the five individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.
```

- **Architect**: Kris Borchers (@kborchers)
- **messageformat**: Eemeli Aro (@eemeli)
- **Mocha**: Christopher Hiller (@boneskull)
- **Node-RED**: Nick O'Leary (@knolleary)
- **WebdriverIO**: Christian Bromann (@bromann)

> - **webhint**: Antón Molleda (@molant)

Reviewer 8: This lists six people. That can't be right?

Author: We ended up with six in the end I believe

Author: Ahhhh

https://github.com/openjs-foundation/cross-project-council/issues/152#issuecomment-494943048

I guess that got added in this PR. @christian-bromann it appears that you unfortunately were not added to the CPC voting members during the election period... was there a conversation or another thread that resulted in you being added to the list?

For now I'm going to remove your name but we can add it back if there is a mistake
